README.md

```diff
@@ -78,6 +78,9 @@ Using express.js:
 
 ```js
 ...
+const MobiusClient = require("@mobius-network/mobius-client-js/lib"); 
+// import MobiusClient from "@mobius-network/mobius-client-js/lib"
```

Comment:
Unfortunately, this is not correct. 

As you can see in example code we use `MobiusClient.Auth.Challenge`. To make it live, we need to import `MobiusClient` via either

```js
const MobiusClient = require("@mobius-network/mobius-client-js"); 
```

or 

```js
import MobiusClient from "@mobius-network/mobius-client-js"
```

--since `lib/index.js` is the main file of NPM package.

Reply:
Well maybe I should create an issue then, cause, I tried that and the app (custom) wasn't importing it

Reply:
Hmm, there is no issue here. This is how ES6 modules works, actually.

As you can see here https://github.com/mobius-network/mobius-client-js/blob/master/src/index.js there is no default export - only named exports.

So if you're working with ES6-compatible code, you should  use it this way:

```js
import { Auth, AppBuilder, Blockchain, Client } from "@mobius-network/mobius-client-js";

// and after that use it like this
Auth.call(...)
AppBuilder.build(...)
```

--or, using CommonJS  format:

```js
const Mobius, { Auth, AppBuilder } = require("@mobius-network/mobius-client-js");

//and after that
Auth.call(...)
// or
Mobius.Auth.call(...)
```

Reply:
I was pretty much using ES6/7 literals.

The problem was actually from the set webpack config.